inc/settings.php

```diff
@@ -105,6 +113,12 @@ class Optml_Settings {
 	 * Optml_Settings constructor.
 	 */
 	public function __construct() {
+
+		if ( self::$is_legacy_install === null ) {
+			// CHANGE THIS WHEN DEPLOYING LIVE
+			self::$is_legacy_install = get_option( 'optimole_wp_install', 0 ) < 1681313471;
```

Comment:
@GrigoreMihai I think there are better solutions than this since not all the users update the plugin at the same time. 

I would suggest just updating the value on some init hook or admin_init, or template_redirect based if the plugin is connected or not.

Reply:
I think that the update time is not affecting this, because we want to know if the plugin was installed before or after the release to have the option enabled/disabled based on that. 

Not sure if you are referring that is not efficient to have this check here and instead we should update the `no_script` option only once on some hook ? (`admin_init` seems the best for this )

Reply:
No, I think this hack with time is  not something reliable and I would like to avoid it.

Reply:
Thank you for the clarification, I get it now.

Reply:
Changed it now if an user is updating the plugin and it is connected it will on by default , for a clean install it will be off.